<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ATMECSINLT-038" failures="0" tests="6" name="com.atmecs.testScripts.InsightsValidation" time="0.000" errors="1" timestamp="22 Oct 2019 07:36:00 GMT" skipped="5">
  <testcase classname="com.atmecs.testScripts.InsightsValidation" name="intitailizeBrowser" time="0.000">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.atmecs.testBase.TestBase.intitailizeBrowser(TestBase.java:76)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
    </error>
  </testcase> <!-- intitailizeBrowser -->
  <testcase classname="com.atmecs.testScripts.InsightsValidation" name="anchorRedirection" time="0.000">
    <skipped/>
  </testcase> <!-- anchorRedirection -->
  <testcase classname="com.atmecs.testScripts.InsightsValidation" name="validateDate" time="0.000">
    <skipped/>
  </testcase> <!-- validateDate -->
  <testcase classname="com.atmecs.testScripts.InsightsValidation" name="validateTitle" time="0.000">
    <skipped/>
  </testcase> <!-- validateTitle -->
  <testcase classname="com.atmecs.testScripts.InsightsValidation" name="validateBreadCrumb" time="0.000">
    <skipped/>
  </testcase> <!-- validateBreadCrumb -->
  <testcase classname="com.atmecs.testScripts.InsightsValidation" name="validateErrorMsg" time="0.000">
    <skipped/>
  </testcase> <!-- validateErrorMsg -->
</testsuite> <!-- com.atmecs.testScripts.InsightsValidation -->
